src/components/home/PanelResult.vue

```diff
@@ -147,6 +251,7 @@ export default Vue.extend({
     ...mapGetters({
       userGenes: 'getUserGenes',
       userGenesInPanels: 'getUserGenesInPanels',
+      institutionsMap: 'getInstitutionsMap'
```

Comment:
Rename (and update references) to institutionMap: 'getInstitutionMap' (no need for the 's')

Reply:
updated